aiida-backup.sh

```diff
@@ -4,15 +4,20 @@
 
 show_help() {
     echo
-    echo "Usage: ./aiida-backup.sh [-h] [-e env-type] [-c conda-env] [-v venv-path] [-p projects-path] [-n project-name]"
+    echo "Usage: ./aiida-backup.sh [-h] [-e env-type] [-c conda-env] [-v venv-path] [-p projects-name] [-n profile-names]"
+    echo
+    echo "Examples:"
+    echo "  conda:            ./aiida-backup.sh -e conda -c <conda-env> -n <project-name>"
+    echo "  venv:             ./aiida-backup.sh -e venv -v aiida2.5 -n <project-name>"
+    echo "  aiida-project:    ./aiida-backup.sh -e aiida-project -n <project-name>"
     echo
     echo "Options:"
     echo "  -h                Show this help message"
     echo "  -e env-type       The type of environment to activate (conda, venv, aiida-project)"
     echo "  -c conda-env      The name of the Conda environment to activate"
     echo "  -v venv-path      The path to the virtual environment to activate"
-    echo "  -n project        The name of the AiiDA project"
-    echo "  -p profiles       The names of AiiDA profiles to backup (optional) - if not specified, all profiles will be backed up"
+    echo "  -n project-name   The name of the AiiDA project. This is not related to aiida-project but a personal choice for your project."
+    echo "  -p profile-names  The names of AiiDA profiles to backup (optional) - if not specified, all profiles will be backed up"
```

Comment:
Hmm, I prefer the shorter names. If you changed it to reflect the above Usage line, note my previous comment.

Reply:
Sure